### DISCUSSION TITLE
Anybody created a GLTF/GLB exporter for StereoKit models?

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#734] [Q&A]
I'm generating mesh objects and converting them to models in StereoKit. I would like to export the model generated along with the material color. For example, creating a cubic shape with some facets in the "faces" of the model and apply a color to the whole model. I would like to export the model to the gltf or ultimately to glb.
Has anybody done this in StereoKit? Any ideas or suggestions are appreciated!

**Comment by Strepto** [comment]
We did a basic GLTF export here, if thats any help?

https://github.com/Strepto/CubeCraft_StereoKit/blob/main/Features/Gltf/GltfExportBox3DModel.cs

**Comment by laultman** [comment]
@Strepto Thanks! Looks very promising. I will see where I can take this. We are creating models programmatically with a node hierarchy of objects. That we would like to export to a GLTF file.
